### DISCUSSION TITLE
Display 2 spaces side-by-side

### DISCUSSION THREAD

**Discussion by gdcondor** [asmvik/yabai#2191] [General]
Hello,
I just discovered Yabai while looking for a solution to display 2 spaces side-by-side under MacOS.
It is possible? Split view or others are not really matching what I am looking to achieve - having the same workflow as 2 separate monitors on a single widescreen.
Or should I consider a different workflow?
Thanks!

**Comment by szfkamil** [comment]
IMO, find a different workflow. I use a single widescreen as well and from experience, I can tell you'll get used to using a single widescreen after switching from a dual-monitor set-up…
